src/Resources/Loaders.cpp

```diff
@@ -0,0 +1,79 @@
+/*****************************************************************************
+ * Copyright (c) 2018-2022 openblack developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/openblack/openblack
+ *
+ * openblack is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#include "Resources/Loaders.h"
+
+namespace openblack::resources
```

Comment:
You can just have `using namespace openblack::resources`. There's nothing new being defined

Reply:
Fixed